### PR TITLE
docs(documentation): Misc documentation improvements

### PR DESCRIPTION
**Pull request by jameswburke**
## Summary
This PR has a bunch of documentation updates and additions. There's no rhyme or reason as to what got updated vs. what will be handled later.

## Notes for reviewers
None.

## Changelog entries
* **Changed**: Addresses applicable feedback in @techmeg's [Notes on Storybook](https://docs.google.com/document/d/1XRxSRxRP-3BJTGKEj6PwSC4OaE1LGhWLtj986UhbLDw/edit#heading=h.3yo2uf10tn1l)
* **Changed**: Updates the environment variables to be more readable.
* **Changed**: Rewrote the Component Mapping and Request Life Cycle sections.

## Ticket(s)
https://alleyinteractive.atlassian.net/browse/IRV-892

### CODE REVIEW

File: stories/02-configuration/04-WordPress.stories.mdx [hunk 1]
```diff
@@ -3,19 +3,94 @@
 # WordPress Configurations
 This section outlines the different ways you can configure WP Irving.
 
-Coming soon.
-
 ## Available Hooks
-This section is a work in progress.
+**Disclaimer: This section is a work in progress. Hooks will rarely be removed, but this list is not yet comprehensive.**
```
Comment (joemcgill) [inline stories/02-configuration/04-WordPress.stories.mdx:7]: We should look at dynamically generating these docs. I'm not sure how much value a list of hooks without usage info is in this context.

File: stories/04-components/06-Component-Mapping.stories.mdx [hunk 2]
```diff
@@ -1,65 +1,118 @@
...
+    "content": "Hello World"
+  }
 }
 ```
 
-## Identifying the name of a React component
+But without mapping it, Irving displays a warning message, `Component "tutorial/button" is not configured.`
 
-The name of the React component in the example below is "Byline." React component names are usually capitalized single words; when then names contain more than one word, they are often camel-cased ('MenuItem').
+<!-- @todo add a screenshot of the warning message -->
```
Comment (joemcgill) [inline stories/04-components/06-Component-Mapping.stories.mdx:63]: Still todo

**Review by joemcgill — APPROVED** [review]
Couple of comments, but these changes look good to me. 👍🏻

File: stories/02-configuration/04-WordPress.stories.mdx (hunk 1 above)
Comment (jameswburke) [reply stories/02-configuration/04-WordPress.stories.mdx:7]: Agreed, I'll create a ticket for it. If anything, it's more of an acknowledgment that they exist and putting up a big flag for us to make sure we come back to it.

**Pull request merged**